### PR TITLE
Add an implementation of SYN flood attack

### PR DESCRIPTION
**Pull request by nadzyah**

### CODE REVIEW

File: hackme/arp_spoofing.py [hunk 1]
```diff
@@ -43,12 +42,13 @@ def __init__(self, interface, your_mac):
...
-            " and MAC-address %s",
-            interface,
-            your_mac,
-        )
+        if interface and your_mac:
+            _LOGGER.debug(
+                "Creating an ARPSpoofer object with your interface %s"
+                " and MAC-address %s",
+                self.interface,
+                self.mac,
```
Comment (nadzyah) [inline hackme/arp_spoofing.py]: `self.mac` should be replaced with `mac`, because `self.mac` is a hex-string

File: hackme/syn_flood.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                send(IP_Packet / TCP_Packet, verbose=0)
+                progress_bar(x, self.count)
+                total += 1
+            except KeyboardInterrupt:
+                _LOGGER.info("\nTotal packets sent: %i\n", total)
+                _LOGGER.warning("\nStopping SYN flood attack")
+                sys.exit(0)
+            except Exception as exc:
+                _LOGGER.error(exc)
+                sys.exit(1)
```
Comment (nadzyah) [inline hackme/syn_flood.py]: We capture it in main

File: hackmeapp.py [hunk 3]
```diff
@@ -113,6 +134,21 @@ def main():
...
+            if args.desc:
+                print(syn_flooder.description)
+                sys.exit(0)
+            if all((args.destIP, args.port, args.count)) is False:
+                parser_syn_flood.print_help()
+                sys.exit(0)
+            syn_flooder.run()
+        except Exception as exc:
+            _LOGGER.error("\n%s", exc)
+
```
Comment (nadzyah) [inline hackmeapp.py:159]: Add `sys.exit(1)`

File: hackmeapp.py [hunk 4]
```diff
@@ -113,6 +134,21 @@ def main():
         except Exception as exc:
             _LOGGER.error(exc)
```
Comment (nadzyah) [inline hackmeapp.py:143]: Add `sys.exit(1)`

File: hackme/syn_flood.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            _LOGGER.debug(
+                "Creating a SYNFlooder object with destination IP %s"
+                " and destination port %i",
+                self.dst_ip,
+                self.dst_port,
+            )
+        self.description = textwrap.dedent(
+            """\
+            The SYN flood attack description:
+            """
```
Comment (nadzyah) [inline hackme/syn_flood.py]: Add a valid description

**Comment by nadzyah** [comment]
The Readme file should be updated

**Review by mdraevich — APPROVED** [review]

**Pull request merged**